javascript/javascript2/week3/scripts/multiPlayer.js

```diff
@@ -0,0 +1,140 @@
+import { questions } from "./questions.js";
+const quizContainer = document.querySelectorAll(".quiz-container");
+const quantity = document.querySelectorAll(".quantity-outof");
+const questionNumber = document.querySelectorAll(".question-number");
+const pointHolder = document.querySelectorAll(".point-holder");
+const questionTitle = document.querySelectorAll(".question-title");
+const buttonsPlayer1 = document.querySelectorAll(".buttons-player1");
+const buttonsPlayer2 = document.querySelectorAll(".buttons-player2");
+
+await whoIsTheWinner();
+
+async function resultAndSwitchPlayer1() {
+  let correctPoints = 0;
+  let incorrectPoints = 0;
+
+  let currentQuestionIndex = 0;
+  let answer;
+
+  while (currentQuestionIndex < questions.length) {
+    const question = questions[currentQuestionIndex];
+
+    questionTitle[0].textContent = question.question;
+    quantity[0].textContent = `${currentQuestionIndex + 1} out of ${
+      questions.length
+    }`;
+    pointHolder[0].textContent = `Correct: ${correctPoints}`;
+    questionNumber[0].textContent = `Incorrect: ${incorrectPoints}`;
+    answer = question.answer;
+
+    let value = await new Promise((resolve) => {
+      buttonsPlayer1.forEach((button) => {
+        button.addEventListener(
+          "click",
+          () => {
+            resolve(button.value === "true");
+          },
+          { once: true }
+        );
+      });
+    });
+
+    if (value === answer) {
+      correctPoints++;
+    } else {
+      incorrectPoints++;
+    }
+
+    currentQuestionIndex++;
+  }
+
+  console.log("Quiz completed!");
+  questionTitle[0].textContent = "Quiz completed!";
+  quantity[0].textContent = "";
+  pointHolder[0].textContent = `Correct: ${correctPoints}`;
+  questionNumber[0].textContent = `Incorrect: ${incorrectPoints}`;
+  return [correctPoints, incorrectPoints];
+}
+
+async function resultAndSwitchPlayer2() {
```

Comment:
You should not have created a separate function for player 2; instead, you should have made the `resultAndSwitchPlayer()` function take a parameter to choose which player's inputs and scores we should use.